package.json

```diff
@@ -70,6 +70,11 @@
         "command": "pw.extension.toggle.showTrace",
         "title": "%contributes.command.pw.extension.toggle.showTrace%"
       },
+      {
```

Comment:
Let's keep it as a hidden setting at first, not visible in a sidebar, no toggle action.